friends/src/App.js

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from "react"
+import axios from "axios"
+
+import "./App.css"
+import FriendsList from "./components/FriendsList"
+
+class App extends Component {
+  constructor() {
+    super()
+    this.state = {
+      friends: []
+    }
+  }
+
+  componentDidMount() {
+    axios
+      .get("http://localhost:5000/friends")
+      .then(res => {
+        this.setState({
+          friends: res.data
+        })
+      })
+      .catch(error => console.log("error"))
```

Comment:
Instead of printing a string, I would use the error that is returned to obtain more info.